akka-persistence-typed/src/main/scala/akka/persistence/typed/internal/Running.scala

```diff
@@ -70,6 +71,8 @@ import akka.util.{ unused, Timeout }
 @InternalApi
 private[akka] object Running {
 
+  private val log = LoggerFactory.getLogger(Running.getClass)
```

Comment:
Use `setup.logger` instead because with this we don't get persistence MDC or custom name? (In my PR I passed setup to `startReplicationStream` instead of `system` and `ref`)

Reply:
at the time i had a log line in a stream operator so that wasn't legal. The log is pretty useless so I'll just remove it in this PR